### PR TITLE
correcting logging description

### PR DESCRIPTION
**Pull request by kalexand-rh**
From https://access.redhat.com/discussions/4232741

You can deploy logging now. https://docs.openshift.com/container-platform/4.1/logging/efk-logging-deploying.html

@mburke5678, if you hear about a method to move the logging components, please let me know.

### CODE REVIEW

File: modules/infrastructure-moving-logging.adoc [hunk 1]
```diff
@@ -7,6 +7,6 @@
 
 [IMPORTANT]
 ====
-The log aggregation solution in {product-title} is not installed by default and
-cannot currently be deployed.
-====
\ No newline at end of file
+The log aggregation solution in {product-title} is not installed by default.
```
Comment (mburke5678) [inline modules/infrastructure-moving-logging.adoc]: Sorry. Should have seen this earlier. The _log aggregation solution_ is now called _cluster logging_.

Comment (kalexand-rh) [reply modules/infrastructure-moving-logging.adoc]: Now is plenty soon enough. :) Do you see anything else that needs to change immediately? If not, I'll merge.

Comment (mburke5678) [reply modules/infrastructure-moving-logging.adoc]: Merge away. Let's at least correct the initial issue. Shall I update that Community Discussions issue?

Comment (kalexand-rh) [reply modules/infrastructure-moving-logging.adoc]: Please do!

**Pull request merged**